website/content/docs/auth/saml.mdx

```diff
@@ -0,0 +1,158 @@
+different path, specify `-path=/my-path` in the CLI.
+
+```shell-session
+$ vault login -method=saml role=admin
+
+Complete the login via your SAML provider. Launching browser to:
+
+    https://company.okta.com/app/vault/abc123eb9xnIfzlaf697/sso/saml?SAMLRequest=fJI9b9swEIZ3%2FwqBu0SJ%2FpBDRAZce4iBtDViN0MX40Sda...
+```
+
+The default browser will open to the generated URL where the user must authenticate with
+the configured SAML Identity Provider. The URL may be manually entered into the browser
+if it cannot be automatically opened.
+
+The CLI login behavior may be customized with the following optional parameters:
+
+- `skip_browser` (default: `false`): If set to `true`, automatic launching of the default
+  browser will be skipped. The SAML Identity Provider URL must be manually entered in a
+  browser to complete the authentication flow.
+- `abort_on_error` (default: `false`): If set to `true`, failure to launch the default
+  browser will result in a non-zero exit of the CLI and an error displayed.
+
+## Configuration
+
+Auth methods must be configured in advance before users or machines can
+authenticate. These steps are usually completed by an operator or configuration
+management tool.
+
+1. Enable the SAML auth method:
+
+   ```shell-session
+   $ vault auth enable saml
+   ```
+
+1. Use the `/config` endpoint to save the configuration of your SAML Identity Provider and
+   set the default role. You can configure the trust relationship with the SAML Identity
+   Provider by either providing a URL for its Metadata document:
+
+   ```shell-session
+   $ vault write auth/saml/config \
+      default_role=admin \
+      idp_metadata_url=https://company.okta.com/app/abc123eb9xnIfzlaf697/sso/saml/metadata \
+      entity_id="https://my.vault/v1/auth/saml" \
+      acs_urls="https://my.vault/v1/auth/saml/callback"
+   ```
+
+   or by setting the configuration Metadata manually:
+
+   ```shell-session
+   $ vault write auth/saml/config \
+      default_role=admin \
+      idp_sso_url=https://company.okta.com/app/abc123eb9xnIfzlaf697/sso/saml \
+      idp_entity_id=https://www.okta.com/abc123eb9xnIfzlaf697 \
+      idp_cert=@path/to/cert.pem \
+      entity_id="https://my.vault/v1/auth/saml" \
+      acs_urls="https://my.vault/v1/auth/saml/callback"
+   ```
+
+1. Create a named role:
+
+   ```shell-session
+    $ vault write auth/saml/role/admin \
+       bound_subjects="*@hashicorp.com" \
+       bound_subjects_type="glob" \
+       token_policies="writer" \
+       bound_attributes=group="admin" \
+       ttl=1h
+   ```
+
+   This role authorizes users that have a subject with an `@hashicorp.com` suffix and
+   are in the `admin` group to authenticate. It also gives the resulting Vault token a
+   time-to-live of 1 hour and the `writer` policy.
+
+   For the complete list of configuration options, please see the
+   [API documentation](/vault/api-docs/auth/saml).
```

Comment:
```suggestion
Refer to full SAML  [API documentation](/vault/api-docs/auth/saml) for a
complete list of configuration options.
```

Referring readers to the API reference is essentially a "next step" suggestion, it's not really part of the previous step, so it doesn't need to be indented. But it might make sense to add a `## Next steps` heading to make it clear they've reached the end of the instructions.